site/public/js/server.js

```diff
@@ -575,7 +575,7 @@ function downloadCSV(code) {
     var temp_element = $('<a id="downloadlink"></a>');
     var address = "data:text/csv;charset=utf-8," + encodeURIComponent(csv_data);
     temp_element.attr('href', address);
-    temp_element.attr('download', 'submitty_user_emails.csv');
+    temp_element.attr('download', 'course_users_data.csv');
```

Comment:
can we make the name here include semester / course details?

Reply:
Yes, added this functionality.